stories/Panel.stories.js

```diff
@@ -0,0 +1,37 @@
+import { storiesOf } from '@storybook/vue';
+import { previewWrapper } from './utils';
+
+storiesOf('Panel', module)
+  .addDecorator(previewWrapper())
+  .add('default', () => ({
+    template: `
+      <ais-panel>
+        This is the body of the Panel.
```

Comment:
there should be some examples with InstantSearch widgets as example I think

Reply:
Each widgets will have an example with the Panel, but it's in the next PR. You can take a look at [Menu](https://github.com/algolia/vue-instantsearch/blob/56e8ddef6c05e9fc7efadc1fa6b0df38ef0e8d24/stories/Menu.stories.js#L50-L57) for example.